public/catalog/v1.0.0/openapi.json

```diff
@@ -1033,6 +1033,58 @@
           }
         }
       }
+    },
+    "/portfolio_items/{portfolio_item_id}/copy": {
+      "post": {
+        "tags": [
+          "PortfolioItem"
+        ],
+        "summary": "Make a copy of the Portfolio Item",
+        "operationId": "postCopyPortfolioItem",
+        "description": "Make a copy of the Portfolio Item.",
+        "parameters": [
+          {
+            "$ref": "#/components/parameters/PortfolioItemID"
```

Comment:
@lindgrenj6 Think of adding an optional name here.

Reply:
Added.